package.json

```diff
@@ -396,6 +396,11 @@
           "default": false,
           "description": "Enable Forge functionality (show pull requests, issues, etc from e.g. Github)"
         },
+        "magit.hide-status-sections": {
+          "type": "array",
+          "default": [],
+          "description": "Array of section names to be hidden in status view. The section names that may be hidden are: 'issues', 'pull requests', 'recent commits', 'staged', 'stashes', 'unmerged', 'unpulled', 'unpushed', 'unstaged', 'untracked'."
```

Comment:
Hard-coding the section names in the help text feels inelegant and threatens to become inaccurate in the future when someone adds a section. However, I couldn't think of a better solution. I think that VSCode doesn't allow us to use the type system beyond declaring it as an array (e.g. we can't define an enum and declare that  the value is of type `StatusSectionName[]`). And I thought it was best not to lead the user to believe that the section names will necessarily be derived from the labels in the UI in any particular way, since that would make you feel less able to change the labels in the UI in the future.